src/js/ui/TxHash/index.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import {getNetwork} from "../../features/network/selectors";
+import {connect} from "react-redux";
+
+const etherScanUrls = {
+  1: 'https://etherscan.io/tx/',
+  3: 'https://ropsten.etherscan.io/tx/',
+  4: 'https://rinkeby.etherscan.io/tx/',
+  5: 'https://goerli.etherscan.io/tx/',
+  42: 'https://kovan.etherscan.io/tx/',
+  401697: 'https://tobalaba.etherscan.com/tx/'
+};
+
+const TxHash = ({value, network}) => {
+  if (!network || (!network.id && network.id !== 0) || !etherScanUrls[network.id]) {
+    return value;
+  }
+
+  return (<a href={etherScanUrls[network.id] + value} target="_blank" rel="noopener noreferrer">{value}</a>);
```

Comment:
What do you think of adding a confirmation that the user is leaving the teller network dapp? Status does not support atm opening links in new tabs.

Reply:
Yeah that's an idea. It makes me think, maybe it supports opening in a new window? Maybe Status is stupid and it doesn't have tabs, but it has windows?
I'll give it a try

Reply:
Mind if I do that in another PR? This one is already complex enough without adding another layer. (gonna add it to Pivotal)

Reply:
np!